### PR TITLE
fixed RoE/ua_abilitycategories.lst filename, warforged issues in RoE/…

### PR DESCRIPTION
**Pull request by digitalseraphim**
…ua_abilities.lsg

### CODE REVIEW

File: data/5e/wizards_of_the_coast/supplement/unearthed_arcana/races_of_eberron/ua_abilities.lst [hunk 1]
```diff
@@ -162,9 +162,9 @@ Light Step				KEY:RoE Warforged ~ Skirmisher ~ Light Step			SORTKEY:1_Race ~ War
...
+Heavy plating (armor)		KEY:RoE Warforged ~ Integrated Protection ~ Heavy plating		PREMULT:2,[!PREABILITY:1,CATEGORY=Special Ability,TYPE.AC_Tracker_Choice],[PREABILITY:1,CATEGORY=Special Ability,RoE Warforged ~ Integrated Protection]		SORTKEY:1_Race ~ Warforged ~ 1i1	PRERACE:1,Warforged	PRETEXT:Heavy armor proficiency	PREABILITY:1,CATEGORY=Internal,Armor Proficiency ~ Heavy	CATEGORY:Special Ability	TYPE:Warforged ~ Integrated Protection.SpecialQuality.Defense.AC_Tracker.AC_Tracker_Choice.AC_Tracker_SelectionMade	DESC:16 + your proficiency bonus; disadvantage on Dexterity (Stealth) checks			BONUS:VAR|AC_Total|AC_Base+Proficiency_Bonus	BONUS:VAR|AC_Base|16|TYPE=Base	BONUS:VAR|AC_Total|AC_Shield|PREVAREQ:AC_ShieldDisabled,0	BONUS:VAR|DisableDexForAC|1|TYPE=Boolean										COST:0
```
Comment (BahamutDragon) [inline data/5e/wizards_of_the_coast/supplement/unearthed_arcana/races_of_eberron/ua_abilities.lst:167]: Why are we breaking the PRExxx tags here to keep the ability normal looking in the UI?

**Review by BahamutDragon — CHANGES_REQUESTED** [review]
Revert the PRExxx change on the plating options, everything else looks proper.

File: data/5e/wizards_of_the_coast/supplement/unearthed_arcana/races_of_eberron/ua_abilities.lst (hunk 1 above)
Comment (digitalseraphim) [reply data/5e/wizards_of_the_coast/supplement/unearthed_arcana/races_of_eberron/ua_abilities.lst:167]: Without that change, the options don't show up. It looks like they depend on themselves. I couldn't choose any of the options when at least the darkwood core option should have been available. With this change and no armor specialization, only darkwood was available. When I added a class that had medium armor, composite showed up.

Comment (BahamutDragon) [reply data/5e/wizards_of_the_coast/supplement/unearthed_arcana/races_of_eberron/ua_abilities.lst:167]: I'll accept this and then deep dive on what is going wrong later. Likely the base ability has a TYPE that's blocking.

Comment (digitalseraphim) [reply data/5e/wizards_of_the_coast/supplement/unearthed_arcana/races_of_eberron/ua_abilities.lst:167]: Maybe I'm confused, but aren't these the specific "instances" of the "integrated protection" ability, and the PREABILITY that I modified is just saying that we have the "base" ability, and then along with the later PRETEXT allows for enabling which versions are allowed based on which armor proficiency you have?

**Pull request merged**